packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -719,6 +750,21 @@ class SliverGeometry with Diagnosticable {
   ///  * [RenderViewport.cacheExtent] for a description of a viewport's cache area.
   final double cacheExtent;
 
+  /// The amount of space allocated to the cross axis.
+  ///
+  /// This value should equal the [crossAxisExtent] of the sliver. If null, then
```

Comment:
Should it? Most cases this will be null. It is only relevant right now for the constrained cross axis sliver.

Reply:
You're right, I'll rephrase this to say that it will be null unless it is different from `SliverConstraints.crossAxisExtent`.